incubator/minecraft/README.md

```diff
@@ -0,0 +1,64 @@
+# Minecraft
+
+[Minecraft](https://minecraft.net/en/) Minecraft is a game about placing blocks and going on adventures.
+
+## Introduction
+
+This chart creates a single Minecraft Pod, plus Services for the Minecraft server and RCON.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`, read the [Minecraft EULA](https://account.mojang.com/documents/minecraft_eula) run:
+
+```bash
+$ helm install --name my-release \
+    --set MinecraftServer.EULA=true incubator/minecraft
```

Comment:
@prydonius I did this with the assumption that the `quote` filter here would stringify it:

https://github.com/kubernetes/charts/pull/154/files#diff-c198687fd5a49f25def79b8ce2933bdeR29

It seems to accept "TRUE" or "true" on the server side. Does the quote filter act like I think it does?